playbook/app/pb_kits/playbook/pb_detail/detail.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbDetail
+    class Detail < ::Playbook::KitBase
```

Comment:
more a question for @jasperfurniss, but do we need the :: before Playbook here? Most of our kit's .rb files do not have it, but i have seen it in a few places like the pagination kit and the form builder

Reply:
@nidaqg thanks for pointing this out and it can be removed. `::` is used for "access[ing] a constant, module, or class defined inside another class or module."

Reply:
It's a fair question! In most cases, it is recommended to use :: when referencing modules or classes that are defined in the top-level namespace, (in this case Playbook is our top-level) or when you want to be explicit about the module hierarchy. However, if there are no conflicting modules or classes in the current context, leaving out the extra `::` is generally acceptable and more concise! 

Either is fine, buuuuut to Nida's point, yes, let's follow our pattern for the time being of leaving out the `::`
128 places have `Playbook::KitBase`
3 places have `::Playbook::KitBase`

A separate cleanup PR could be made for the 3 places at a later date and tested.